NuRadioReco/framework/base_station.py

```diff
@@ -186,8 +186,11 @@ def is_neutrino(self):
 
     def is_cosmic_ray(self):
         if self._particle_type == '':
-            logger.error("Stations particle type has not been set")
-            raise ValueError("Stations particle type has not been set")
+            msg = "Stations particle type has not been set. Please call the module `eventTypeIdentifier.run(event, station, mode='forced', forced_event_type='neutrino'/'cosmic_ray')`."
+            msg += " This flag is used to differentiate between signals that originate from air vs. signals that originate from within the ice which is needed to e.g. determine if refraction "
+            msg += "into the ice needs to be considered."
+            logger.error(msg)
+            raise ValueError(msg)
```

Comment:
If we add this to the `is_cosmic_ray` method, shouldn't we maybe add it to the `is_neutrino` method too?
It also is very detailed for an error message (maybe you can copy it over to the docstring, too?), but I guess there's no such thing as a too helpful error message : )